src/kili/adapters/kili_api_gateway/organization/mappers.py

```diff
@@ -0,0 +1,60 @@
+"""Mappers for Organization API calls."""
+
+import json
+from typing import Dict
+
+from kili.adapters.kili_api_gateway.organization.types import (
+    KiliAPIGateWayCreateOrganizationInput,
+    KiliAPIGateWayUpdateOrganizationInput,
+)
+from kili.domain.organization import (
+    OrganizationFilters,
+    OrganizationId,
+    OrganizationMetricsFilters,
+)
+
+
+def map_organization_data(data: KiliAPIGateWayCreateOrganizationInput) -> Dict:
+    """Build the GraphQL OrganizationData variable to be sent in an operation."""
+    return {
+        "data": {
+            "name": data.name,
+            "address": data.address,
+            "city": data.city,
+            "country": data.country,
+            "zipCode": data.zip_code,
+        }
+    }
```

Comment:
in other mappers, we don't put the "data" not the "where" keys. They are created in the gateway method directly, not in the mapper.